.github/workflows/test_cpp.yml

```diff
@@ -224,27 +239,33 @@ jobs:
         include:
           - name: C++14
             flags: -DCMAKE_CXX_STANDARD=14
+            run-on-presubmit: true
           - name: C++17
             flags: -DCMAKE_CXX_STANDARD=17
+            run-on-presubmit: false
           - name: C++20
             flags: -DCMAKE_CXX_STANDARD=20
+            run-on-presubmit: true
 
-    name: Linux CMake GCC ${{ matrix.name }}
+    name: ${{ (!matrix.run-on-presubmit && inputs.test-type == 'presubmit') && '[SKIPPED]' || '' }} Linux CMake GCC ${{ matrix.name }} ${{ !matrix.run-on-presubmit && '(Continuous)' || '' }}
```

Comment:
This seems like a lot of sensitive logic we're duplicating many times.  Is it possible to reduce this a little to make it harder to mess up?  Notably, we might want to centralize the spelling of `'[SKIPPED]'` and `'(Continuous)'`.  If we put them both on the same side we might be able to reduce the number of branches too.

e.g. something like `${{ matrix.skip-on-presubmit && inputs.continuous-prefix }}`, and then the runner could compute the prefix in 1 place

Reply:
Makes sense, this is pretty minimal still